db/migrate/20170405094953_re_create_escorts.rb

```diff
@@ -0,0 +1,9 @@
+class ReCreateEscorts < ActiveRecord::Migration[5.0]
+  def change
+    create_table :escorts, id: :uuid do |t|
+      t.string :prison_number
```

Comment:
Should we remove the prison number from the detainee table and just delegate to the escorts prison number for now?